09-coap/test_spec09.py

```diff
@@ -126,8 +91,8 @@ def test_task01(riot_ctrl, log_nodes):
         stderr=None if log_nodes else subprocess.DEVNULL,
     )
     try:
-        res = node.cord_ep_register(HOST_ULA)
-        parser = CordEPRegisterParser()
+        res = node.cord_ep_register("[{}]".format(HOST_ULA))
+        parser = CordEpRegistrationInfoParser()
         core_reg = parser.parse(res)
```

Comment:
Just noticed: as you return `None` in the upstream parser, you need to check here, that it is actually a dictionary you can subscribe `"ltime"` from.
```suggestion
        core_reg = parser.parse(res)
        assert core_reg
```

You can squash that change directly.

Reply:
(locally I get a pylint error otherwise)

Reply:
Actually this doesn't help either :-/. @aabadie do you have an idea what `pylint` is trying to say here?

```
************* Module 09-coap.test_spec09
09-coap/test_spec09.py:98:11: E1136: Value 'core_reg' is unsubscriptable (unsubscriptable-object)
09-coap/test_spec09.py:101:33: E1136: Value 'core_reg' is unsubscriptable (unsubscriptable-object)
09-coap/test_spec09.py:102:19: E1136: Value 'core_reg' is unsubscriptable (unsubscriptable-object)
```

Reply:
As an alternative: let's just fix my review slip-up upstream ^^ https://github.com/RIOT-OS/RIOT/pull/14719

Reply:
With https://github.com/RIOT-OS/RIOT/pull/14719 this is resolved.